.github/actions/vulnerability_scanner_deps/action.yml

```diff
@@ -4,15 +4,30 @@ description: "Download and compiles the dependencies for the vulnerability scann
 runs:
   using: "composite"
   steps:    
-      - name: External deps
+      - name: Dependencies for local execution
+        if: env.ACT # Only run for local execution
+        shell: bash
+        run: |
+
+          # Obtain a copy of the signing key
+          wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null
+          
+          # Add the repository to your sources list
+          echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ jammy main' | sudo tee /etc/apt/sources.list.d/kitware.list >/dev/null
+
+          # Update packages
+          sudo apt-get update
+          sudo apt-get install -y cmake
```

Comment:
Question: Is this cacheable? I saw you are using the cache for `act` executions on `.github/actions/coverage/action.yml` for `bc` package but not here.

Apparently, `act` does support cache: https://github.com/nektos/act/pull/1770

Reply:
And why the difference with `bc`? It would be correct to have the same install method for both packages, IF possible.

Reply:
I will try it out

Reply:
Thanks for the try!